cli/common/utils.go

```diff
@@ -15,18 +15,86 @@ import (
 	"time"
 
 	"github.com/adam-hanna/arrayOperations"
+	"github.com/alecthomas/participle/v2"
+	"github.com/alecthomas/participle/v2/lexer/stateful"
 	"github.com/mitchellh/mapstructure"
 	"github.com/tarantool/cartridge-cli/cli/context"
 	"github.com/vmihailenco/msgpack/v5"
 	"gopkg.in/yaml.v2"
 )
 
+type DepRelation struct {
+	Relation string `@( "=" "=" | "=" | ">" "=" | "<" "=" | ">" | "<" )`
+	Version  string `@Number`
+}
+
+type PackDependency struct {
+	Name      string        `@Ident`
+	Relations []DepRelation `(@@ ( "," @@ )?)?`
+}
+
+type PackDependencies []PackDependency
+
 var bufSize int64 = 10000
 
 func init() {
 	rand.Seed(time.Now().UTC().UnixNano())
 }
 
+func (deps PackDependencies) FormatDeb() PackDependencies {
+	debDeps := make(PackDependencies, 0, len(deps))
+
+	for _, dependency := range deps {
+		for i, r := range dependency.Relations {
+			if r.Relation == ">" || r.Relation == "<" {
+				// Deb format uses >> and << instead of > and <
+				dependency.Relations[i].Relation = fmt.Sprintf("%s%s", r.Relation, r.Relation)
+			} else if r.Relation == "==" {
+				dependency.Relations[i].Relation = "="
+			}
+		}
+
+		debDeps = append(debDeps, dependency)
+	}
+
+	return debDeps
+}
+
+func (deps PackDependencies) FormatRPM() PackDependencies {
+	// We can't get constants from rpm package - cycle imports not allowed.
```

Comment:
Yes, this looks bad.

Seems that `rpm` package can call some function that accepts `deps PackDependencies` and converts them directly to `rpmTagType`.